api_feature.go

```diff
@@ -96,6 +97,11 @@ func (f *APIFeature) IPut(path string, body *godog.DocString) error {
 	return f.makeRequest("PUT", path, []byte(body.Content))
 }
 
+// IPatch makes a PATCH request to the provided path with the current headers and the body provided
+func (f *APIFeature) IPatch(path string, body *godog.DocString) error {
+	return f.makeRequest("PATCH", path, []byte(body.Content))
```

Comment:
What happens if `body == nil`? Is that even possible? (I don't know enough about what's going on here)

Reply:
We should probably ask the same question for the other functions